php-binance-api.php

```diff
@@ -240,9 +243,9 @@ protected function setupProxyConfigFromFile(string $file = null)
      * @param $flags array addtional options for order type
      * @return array with error message or the order details
      */
-    public function buy(string $symbol, $quantity, $price, string $type = "LIMIT", array $flags = [])
+    public function buy(string $symbol, $quantity, $price, string $type = "LIMIT", array $flags = [], $futures = false)
```

Comment:
I think it would be better to use a new class `FAPI` or `FuturesAPI` then to do this with switches. Otherwise, if Margin API is introduced, do you want to be switching Margin **and** Futures on and off?